chia/daemon/keychain_proxy.py

```diff
@@ -109,7 +109,7 @@ async def connect_to_keychain(self) -> None:
                     self._uri,
                     autoclose=True,
                     autoping=True,
-                    heartbeat=60,
+                    heartbeat=self.heartbeat,
```

Comment:
Should add a heartbeat parameter to `.__init__()` to pass through to the `super().__init__()` call.